.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.5
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: install dependencies
+          command: |
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
```

Comment:
you could use gemfile.lock as a cachekey to cache deps, see https://github.com/nedap/StaffingSolutions/issues/44#issuecomment-551891381

Reply:
Yeah, I saw all that code... and I removed everything that seemed unnecessary

Reply:
why did caching of the deps seem unnecessary?

Reply:
+1 on keeping the dependency caching. It speeds up builds significantly, and is automatically invalidated when you need it to.

Even when it invalidates, it will still build on top of an older cache, so it only needs to fetch updated gems instead of all gems. Pretty neat.

Reply:
it seemed unnecessary because of the small scope of these specs and that I don't completely understand it. Isn't your first move to remove everything you don't understand as well? that simplifies things.